### PR TITLE
Mho cbonsai

### PR DESCRIPTION
**Pull request by mhzawadi**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Import from https://github.com/mhzawadi/homebrew-cbonsai

### CODE REVIEW

File: Formula/cbonsai.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  desc "Console Bonsai is a bonsai tree generator, written in C using ncurses"
+  homepage "https://github.com/mhzawadi/homebrew-cbonsai"
+  url "https://github.com/mhzawadi/homebrew-cbonsai/archive/refs/tags/v1.3.4.tar.gz"
+  sha256 "d984f50afab06c5160b7f8577672397203d9d52f6857648c56f0fca4a409fb51"
+  license "GPL-3.0"
+  depends_on "ncurses"
+  depends_on "pkg-config"
+
+  def install
+    ENV["PKG_CONFIG_PATH"] = "$PKG_CONFIG_PATH:/usr/local/Cellar/ncurses/6.4/lib/pkgconfig/"
```
Comment (SMillerDev) [inline Formula/cbonsai.rb:11]: This is done by the dependency

File: Formula/cbonsai.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    ENV.append_path "PKG_CONFIG_PATH", "/opt/homebrew/Cellar/ncurses/6.4/lib/pkgconfig/"
+    system "make"
+    bin.install "cbonsai"
+  end
+
+  test do
+    system bin/"cbonsai", "-p"
```
Comment (SMillerDev) [inline Formula/cbonsai.rb:18]: Can we ensure some output here?

File: Formula/cbonsai.rb (hunk 1 above)
Comment (mhzawadi) [reply Formula/cbonsai.rb:11]: Oh, I had to add it to make it all work. will test and fix

File: Formula/cbonsai.rb (hunk 2 above)
Comment (mhzawadi) [reply Formula/cbonsai.rb:18]: umm, would the help be a better output as that can be tested.

as the output from the `-p` option is a tree that will change each time it runs
image

**Comment by p-linnane** [comment]
From our [Acceptable Formulae](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff) documentation:
>We frown on authors submitting their own work unless it is very popular.

Based off the GitHub statistics, this does not meet this criteria. It also does not meet our notability requirements:
><30 forks, <30 watchers and <75 stars

**Comment by osalbahr** [comment]
cbonsai is originally hosted at https://gitlab.com/jallbrit/cbonsai, btw. Would gitlab statistics suffice the requirements?

File: Formula/cbonsai.rb (hunk 2 above)
Comment (osalbahr) [reply Formula/cbonsai.rb:18]: There are probably flags to make the tree fixed

File: Formula/cbonsai.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class Cbonsai < Formula
+  desc "Console Bonsai is a bonsai tree generator, written in C using ncurses"
+  homepage "https://github.com/mhzawadi/homebrew-cbonsai"
+  url "https://github.com/mhzawadi/homebrew-cbonsai/archive/refs/tags/v1.3.4.tar.gz"
```
Comment (osalbahr) [inline Formula/cbonsai.rb:4]: Would it be possible to directly use the gitlab link instead?

**Comment by Bo98** [comment]
> cbonsai is originally hosted at https://gitlab.com/jallbrit/cbonsai, btw. Would gitlab statistics suffice the requirements?

Yes, but we would use that upstream. It's not clear why it's been forked and updated independently on GitHub.

**Comment by osalbahr** [comment]
Just to clarify, if the formula used https://gitlab.com/jallbrit/cbonsai/-/tree/v1.3.1 instead of the github link then it would satisfy the Acceptable Formula requirements?

**Comment by Bo98** [comment]
Yes, at least on the notability aspect.

This assumes it doesn't require a load of patches etc to work.

**Pull request closed without merging**